src/components/ActionCard.tsx

```diff
@@ -92,23 +51,62 @@ const ActionCard: React.FC<IProps> = ({
   const isImageURL = typeof image === "string"
 
   return (
-    <Card to={to} className={className} hideArrow={true}>
-      <ImageWrapper
-        isRight={isRight}
-        isBottom={isBottom}
+    <LinkBox
```

Comment:
**question:** how will LinkBox handle a case of multiple LinkOverlays with distinct href(s)? Like, I as a user hover over the action card, which URL will the user interact with?

Reply:
(Ignore the above) I realised that you will be using LinkOverlay only if you wanted to have the entire component point to a single resource.